core/PRP/Physics/plGenericPhysical.cpp

```diff
@@ -547,10 +547,31 @@ void plGenericPhysical::IReadPXPhysical(hsStream* S, plResManager* mgr) {
     } else if (fBounds == plSimDefs::kBoxBounds) {
         fDimensions.read(S);
         fOffset.read(S);
-    } else if (fBounds == plSimDefs::kHullBounds) {
-        PXCookedData::readConvexMesh(S, this);
-    } else {    // Proxy or Explicit
-        PXCookedData::readTriangleMesh(S, this);
+    } else {
+        char tag[4];
+        S->read(4, tag);
+
+        if (memcmp(tag, "NXS\x01", 4) == 0) {
+            // Cooked PhysX stream
+            if (fBounds == plSimDefs::kHullBounds)
```

Comment:
Do we need to rewind the stream by 4 bytes before reading the cooked meshes here?
EDIT: Nevermind, I see you deleted the header read below.  Leaving this comment in to remind myself.